packages/loaders/src/utils/spinner-coordinates.js

```diff
@@ -0,0 +1,206 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+export const dPathFrames = [
```

Comment:
In the case of this brute force method, wouldn't it be cleaner/more elegant to extract the keyframe values from https://github.com/zendeskgarden/css-components/blob/jzempel/loaders/packages/loaders/src/_keyframes.css#L92-L148 and apply them to a `<circle>` element (like this SVG: https://github.com/zendeskgarden/css-components/blob/jzempel/loaders/packages/loaders/src/svg/spinner.svg)? In order to work for IE11 the values would need to be applied to attributes of the `<circle>` (rather than CSS properties). I think you'd end up with tighter, simpler to understand code.

Reply:
p.s. take a look at https://github.com/zendeskgarden/css-components/blob/jzempel/loaders/demo/loaders/index.js#L103-L116 to see how I converted the values you're using below to the keyframe data seen in the `css-components` branch.

Reply:
of course 🤦‍♂️  that'll shrink it a bit!